src/FSharp.Data.GraphQL.Server.AspNetCore/GraphQLRequest.fs

```diff
@@ -0,0 +1,252 @@
+namespace FSharp.Data.GraphQL.Server.AspNetCore
+
+open System
+open System.IO
+open System.Text.Json
+open System.Text.Json.Serialization
+open System.Threading.Tasks
+open Microsoft.AspNetCore.Http
+open Microsoft.Extensions.Logging
+open Microsoft.Extensions.Options
+
+open FSharp.Data.GraphQL
+open FsToolkit.ErrorHandling
+
+open FSharp.Data.GraphQL.Server
+
+/// Provides logic to parse and execute GraphQL request
+type GraphQLRequest<'Root> (
```

Comment:
`GraphQLRequest` as a name sounds too "static" and passive, like an HTTP request, which is an object containing information about a request. How about `GraphQLRequestHandler` or `GraphQLRequestHandlingLogic`?

Reply:
Renamed